dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/pageload-update-txn-name/subject.js

```diff
@@ -1,3 +1,3 @@
 const activeSpan = Sentry.getActiveSpan();
 const rootSpan = activeSpan && Sentry.getRootSpan(activeSpan);
-rootSpan?.updateName('new name');
+rootSpan.updateName('new name');
```

Comment:
is this on purpose? It is technically optional, isn't it?

Reply:
or it should be `!.`?

Reply:
Ah right, I think I was just debugging this test. but I think there should be no difference. If the span isn't defined here, we have bigger problems and the test would fail.